File: Blog-Server/src/main/java/com/blog/repository/BlogMapper.java
package com.blog.repository;

import com.blog.pojo.Blog;
import tk.mybatis.mapper.common.Mapper;

/**
 * @author JeungNyeongJae
 * @date 2020/1/6
 */
@org.apache.ibatis.annotations.Mapper
public interface BlogMapper extends Mapper<Blog> {
}

File: Blog-web-service/src/main/java/com/blog/dao/BlogMapper.kt
package com.blog.dao

import com.blog.pojo.Blog
import tk.mybatis.mapper.common.Mapper

/**
 *  @author JeungNyeongJae
 *  @date 2019/11/7
 *
 */
@org.apache.ibatis.annotations.Mapper
interface BlogMapper : Mapper<Blog>
